### DISCUSSION TITLE
Add an explanation and link to the material design icons below whereever it asks for an icon - including the icon field?

### DISCUSSION THREAD

**Discussion by sxsws** [directus/directus#13312] [Feature Requests]
Small nit-picky thing, but this will allow a bigger preview and to search them all full-page rather than in one small field. I think the material design icons aren't named as helpfully as say, font awesome icons are so finding appropriate icons can be a bit awkward.

https://fonts.google.com/icons?selected=Material+Icons

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
